docs/_Aug_milestone_.md

```diff
@@ -0,0 +1,46 @@
+## August Milestone
+
+### Overview
+
+- Finish hasManyThrough relation MVP
+- Continue with docs improvements
+
+### Goals
+
```

Comment:
I think we should start working on CLA-to-DCO switch in August. I expect the task does not require too much work but it will probably span across longer time period. It's better to start it early to have a chance to get it done in Q3.

Link to our internal task: https://github.com/strongloop-internal/scrum-apex/issues/436